src/screen.cpp

```diff
@@ -1153,7 +1153,7 @@ void s_reset(screen_t *s, screen_reset_mode_t mode) {
         if (screen_width > non_space_width) {
             bool justgrey = true;
             if (cur_term && enter_dim_mode) {
-                std::string dim = tparm((char *)enter_dim_mode);
+                std::string dim = tparm(enter_dim_mode);
```

Comment:
This cast is necessary on some platforms. Specifically I'm pretty sure this breaks the build on NetBSD.